### DISCUSSION TITLE
Check Other User Online Status

### DISCUSSION THREAD

**Discussion by 0xJWLabs** [diamondburned/arikawa#454] [Help]
Hey guys, how to check other user status? if they are online, offline, dnd etc? Thanks!

**Comment by hi117** [comment]
This gets sent via PRESSENCE_UPDATE events from the gateway. https://discord.com/developers/docs/topics/gateway-events#presence-update . So you will need to enable the presence privildged intent for your bot and also connect with that intent to the gateway. After that, you can use the State portion of arikawa to track and query for member presences. https://pkg.go.dev/github.com/diamondburned/arikawa/v3@v3.4.0/state#State.Presence
